portfolio/src/main/webapp/script.js

```diff
@@ -51,8 +54,20 @@ const handleClickProject = function() {
     });
 }
 
-function getRandomQuoteUsingArrowFunctions() {
-    fetch('/data').then(response => response.text()).then((quote) => {
-        document.getElementById('quote-container').innerHTML = quote;
+function getComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const commentsContainer = document.getElementById('comments-container');
+        commentsContainer.innerHTML = '';
+        
+        for(let i in comments) {
```

Comment:
You almost always want "of" instead of "in" e.g.

`for (let i of comments) {`

However this means that `i` will be the actual string comment, not an index. To get an index, you'll need to do the usual `for (let i = 0; i < comments.length; i++) {` formula.